lib/enums/taskoptions.dart

```diff
@@ -1,2 +1,2 @@
 // Enum to represent options in the menu
-enum TaskOption { deleteAll }
\ No newline at end of file
+enum TaskOption { deleteAll,showKudos }
```

Comment:
Please add EOF here

Reply:
Sorry but I am unable to understand meaning of "Adding an EOF"  for this.